### DISCUSSION TITLE
Date and Time Format for US Version

### DISCUSSION THREAD

**Discussion by craigs66** [Blackymas/NSPanel_HA_Blueprint#357] [Q&A]
Is it possible to change the date format to mmddyy and the time format from 24hr to 12hr?

Love the US version of the panel!!

**Comment by Blackymas** [comment]
You have to change this manually in the Blueprint.

Open the bleuprint via file editor and change all lines with

`  ##### MUI Multilingual User Interface (DE/EN) #####
  date01_mui: >-
    {%- if language == "DEU" -%} {{ ((as_timestamp(now())) | timestamp_custom('%a %d.%m' )) | replace('Mon', 'Montag,')| replace('Tue', 'Dienstag,')| replace('Wed', 'Mittwoch,')| replace('Thu', 'Donnerstag,')| replace('Fri', 'Freitag,')| replace('Sat', 'Samstag,')| replace('Sun', 'Sonntag,') }}
    {%- elif language == "ENG" -%} {{ ((as_timestamp(now())) | timestamp_custom('%a %d.%m' )) `
    
    
 do not forget relaod automation

**Comment by craigs66** [comment]
For others that find this post, here is what I changed in the Blueprint to get the attached image. Hope it helps!

original:
date01_mui: '{%- if language == "DEU" -%} {{ ((as_timestamp(now())) | timestamp_custom(''%a
    %d.%m'' )) | replace(''Mon'', ''Montag,'')| replace(''Tue'', ''Dienstag,'')| replace(''Wed'',
    ''Mittwoch,'')| replace(''Thu'', ''Donnerstag,'')| replace(''Fri'', ''Freitag,'')|
    replace(''Sat'', ''Samstag,'')| replace(''Sun'', ''Sonntag,'') }} {%- elif language
    == "ENG" -%} {{ ((as_timestamp(now())) | timestamp_custom(**''%a %d.%m''** )) | replace(''Mon'',
    ''Monday,'')| replace(''Tue'', ''Tuesday,'')| replace(''Wed'', ''Wednesday,'')|
    replace(''Thu'', ''Thursday,'')| replace(''Fri'', ''Friday,'')| replace(''Sat'',
    ''Saturday,'')| replace(''Sun'', ''Sunday,'') }}

my version:
date01_mui: '{%- if language == "DEU" -%} {{ ((as_timestamp(now())) | timestamp_custom(''%a
    %d.%m'' )) | replace(''Mon'', ''Montag,'')| replace(''Tue'', ''Dienstag,'')| replace(''Wed'',
    ''Mittwoch,'')| replace(''Thu'', ''Donnerstag,'')| replace(''Fri'', ''Freitag,'')|
    replace(''Sat'', ''Samstag,'')| replace(''Sun'', ''Sonntag,'') }} {%- elif language
    == "ENG" -%} {{ ((as_timestamp(now())) | timestamp_custom(''**%B %d, %Y**'' )) | replace(''Mon'',
    ''Monday,'')| replace(''Tue'', ''Tuesday,'')| replace(''Wed'', ''Wednesday,'')|
    replace(''Thu'', ''Thursday,'')| replace(''Fri'', ''Friday,'')| replace(''Sat'',
    ''Saturday,'')| replace(''Sun'', ''Sunday,'') }} 


original:
time: '{{ (as_timestamp(now()) | timestamp_custom(''%H:%M'')) }}'
my version:
time: '{{ (as_timestamp(now()) | timestamp_custom(''%I:%M%p'')) }}'

![IMG_4584](https://user-images.githubusercontent.com/47399804/220184562-a5a56a32-75d5-4dcd-801f-95bb7c1a90fd.jpg)

**Reply by T3mplate-wow** [reply]
thanks for sharing the solution!
I also looking forward to the us version. But to be honest there is no really change or usage of the bigger screen. Right? 
Or are there some improvements because of the bigger screen?

**Reply by edwardtfn** [reply]
The screen is not really bigger, it's just in a different position, although in the EU version some fo the pixels are not visible.
Take a look at this: #713

**Comment by edwardtfn** [comment]
I'm closing this discussion as we have released v4.0 (#1057).
Please add a feature request for the missing things.

And thanks a lot for your support.
